package.json

```diff
@@ -76,6 +76,7 @@
     "elliptic": "^6.5.4",
     "bn.js": "4.12.0",
     "ejs": "^3.1.7",
+    "fast-json-patch": "^3.1.1",
```

Comment:
This is a resolution

Reply:
yes, but just updating version in DPP packages not enough?

Reply:
I don't understand what you are asking or what change you are proposing. The change is in the monorepo root `package.json` because this is the only legal location for resolutions.